backend/api/management/commands/generate_fixtures.py

```diff
@@ -12,10 +12,30 @@ class Command(BaseCommand):
     help = "Generate fixtures"
 
     def handle(self, *args, **options):
-        simple_algo = factory.create_algo(name="Simple algo", category=Algo.Category.ALGO_SIMPLE)
-        aggregate_algo = factory.create_algo(name="Aggregate algo", category=Algo.Category.ALGO_AGGREGATE)
-        composite_algo = factory.create_algo(name="Composite algo", category=Algo.Category.ALGO_COMPOSITE)
-        metric_algo = factory.create_algo(name="Metric algo", category=Algo.Category.ALGO_METRIC)
+        simple_algo = factory.create_algo(
+            inputs=factory.get_algo_inputs(["datasamples", "opener", "model"]),
+            outputs=factory.get_algo_outputs(["model"]),
+            category=Algo.Category.ALGO_SIMPLE,
+            name="simple algo",
+        )
```

Comment:
Side improvement:
Algo inputs/outputs are provided to the factory instead of being inferred. It removes adherence between test factory  and category.